vite-project/src/scouter/ScoutingTab.tsx

```diff
@@ -50,15 +48,18 @@ function ScouterApp() {
       <MapQuery
         name="CRESCENDO"
         width={640}
```

Comment:
consider moving values to const
(also next line)

Reply:
this file is where you config everything